blobmodel/blobs.py

```diff
@@ -172,15 +169,8 @@ def _propagation_direction_shape(
             )
         else:
             x_diffs = x - self._prop_dir_blob_position(t)
-
-        if self.blob_shape == "gauss":
-            return 1 / np.sqrt(np.pi) * np.exp(-(x_diffs**2 / self.width_prop**2))
-        elif self.blob_shape == "exp":
-            return np.exp(x_diffs / self.width_prop) * np.heaviside(-1.0 * (x_diffs), 1)
-        else:
-            raise NotImplementedError(
-                f"{self.__class__.__name__}.blob_shape not implemented"
-            )
+        theta_x = x_diffs / self.width_prop
+        return self.blob_shape.get_pulse_shape_prop(theta_x, ...)
```

Comment:
... serves as a placeholder for `kwargs `right?

Reply:
yep